src/main/java/org/springframework/pipes/module/launcher/ModuleLauncher.java

```diff
@@ -64,7 +67,7 @@ private static void launchModules(File moduleHome, String... modules) {
 		Executor executor = Executors.newFixedThreadPool(modules.length);
 		for (String module : modules) {
 			module = (module.endsWith(".jar")) ? module : module + ".jar";
-			executor.execute(new ModuleLaunchTask(new File(moduleHome, module)));
+			executor.execute(new ModuleLaunchTask(new File(moduleHome, module), module));
```

Comment:
might be better to move the File ctor call inside ModuleLaunchTask so it just takes 2 params, moduleHome (a File) and the module name String... or something else that reduces the feeling of duplication this has